nautobot/core/management/commands/pre_migrate.py

```diff
@@ -75,6 +86,115 @@ def check_configcontext_uniqueness():
         )
 
 
+def check_permissions_constraints(command):
+    """
+    Check for permission constraints that are referencing an object field
+    that will be migrated to a different model/field/value in 2.0 migrations.
+
+    Objects that will be deleted in 2.0 are:
+    - dcim.DeviceRole
+    - dcim.RackRole
+    - ipam.Aggregate
+    - ipam.Role
```

Comment:
What about Site and Region?

Reply:
I thought we covered those in the migrations but if not I'll add them here